unit_test/test_spdm_requester/get_version.c

```diff
@@ -542,7 +538,7 @@ void libspdm_test_requester_get_version_case8(void **state)
     spdm_test_context->case_id = 0x8;
 
     status = libspdm_get_version(spdm_context, NULL, NULL);
-    assert_int_equal(status, RETURN_DEVICE_ERROR);
+    assert_int_equal(status, LIBSPDM_STATUS_INVALID_MSG_FIELD);
 }
 
 /**
```

Comment:
@jyao1 We can get rid of Test 9, correct? It doesn't do what it says it does and is effectively the same as Test 8.